sdk/core/azure-core/azure/core/pipeline/transport/_base.py

```diff
@@ -592,6 +594,15 @@ def raise_for_status(self):
         if self.status_code >= 400:
             raise HttpResponseError(response=self)
 
+    def __repr__(self):
+        # there doesn't have to be a content type
+        content_type_str = (
```

Comment:
Nitpick: I would exclude the content type key/value altogether in the repr if we don't know what it is (i.e. not add `, Content-Type: ` to the output...